packages/components/tabs/__tests__/tabs.test.tsx

```diff
@@ -8,6 +8,17 @@ import TabNav from '../src/tab-nav'
 import type { TabPaneName } from '../src/tabs'
 import type { TabsPaneContext } from '@element-plus/tokens'
 
+const Comp = defineComponent({
+  components: {
+    TabPane,
+  },
+  template: `
+  <TabPane name="tab1" label="tab1">
+    Tab 1 content
+  </TabPane>
+    `,
+})
+
```

Comment:
## Question

I have a very trivial question. Is there any special reason to use `template` syntax in `tsx` files?

Reply:
My mistake, a way of writing I saw recently, and I wrote it subconsciously.